File: Baekjoon/python/25828.py
# https://www.acmicpc.net/problem/25828

def main():
    # input
    g, p, t = map(int, input().split())

    individual, gruop = g * p, g + p * t

    if individual > gruop:
        print(2)
    elif individual < gruop:
        print(1)
    else:
        print(0)

if __name__ == '__main__':
    main()

File: Baekjoon/go/25828.go
// https://www.acmicpc.net/problem/25828

package main

import (
	"fmt"
)

func main() {
	var g, p, t int

	// input
	fmt.Scan(&g, &p, &t)

	individual := g * p
	group := g + p * t

	if individual > group {
		fmt.Println(2)
	} else if individual < group {
		fmt.Println(1)
	} else {
		fmt.Println(0)
	}
}